### PR TITLE
Add external Id and cloudformation template url endpoints

### PR DESCRIPTION
**Pull request by meln1k**
# Description

# To-Dos

- [x] Add test coverage for new or updated functionality
- [x] Lint and test with `nox`

# Code of Conduct

By submitting this pull request, I agree to follow the [code of conduct](https://some.engineering/code-of-conduct).

### CODE REVIEW

File: fixbackend/organizations/router.py [hunk 1]
```diff
@@ -167,4 +167,26 @@ async def accept_invitation(
 
         return None
 
+    @router.get("/{organization_id}/external_id")
+    async def get_externa_id(
```
Comment (aquamatthias) [inline fixbackend/organizations/router.py:171]:
```suggestion
    async def get_external_id(
```

File: migrations/versions/2023-09-20T09:32:21Z_add_externalid_to_org.py [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+# revision identifiers, used by Alembic.
+revision: str = "aba43062e638"
+down_revision: Union[str, None] = "9bd6d9dc6957"
+branch_labels: Union[str, Sequence[str], None] = None
+depends_on: Union[str, Sequence[str], None] = None
+
+
+def upgrade() -> None:
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column("organization", sa.Column("external_id", GUID, nullable=False, server_default=sa.text("uuid()")))
```
Comment (aquamatthias) [inline migrations/versions/2023-09-20T09:32:21Z_add_externalid_to_org.py:24]: looks like the wrong default or? Is that something we can invent?

File: migrations/versions/2023-09-20T09:32:21Z_add_externalid_to_org.py [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+    # ### end Alembic commands ###
+
+
+def downgrade() -> None:
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.drop_column("organization", "external_id")
```
Comment (aquamatthias) [inline migrations/versions/2023-09-20T09:32:21Z_add_externalid_to_org.py:30]: would we ever drop columns when going back? Shouldn't the create statement be more clever and ignore existing columns?

File: fixbackend/organizations/router.py [hunk 4]
```diff
@@ -167,4 +167,26 @@ async def accept_invitation(
...
+            raise HTTPException(
+                status_code=403, detail="You must be an owner of this organization to get an external ID"
+            )
+
+        return ExternalId(external_id=org.external_id)
+
+    @router.get("/{organization_id}/cf_url")
+    async def get_cf_url(organization_id: UUID, user_context: AuthenticatedUser) -> str:
+        return "https://example.com"
+
```
Comment (aquamatthias) [inline fixbackend/organizations/router.py:191]: Don't we need an endpoint to get account id, name, external_id and role_arn?

File: fixbackend/organizations/router.py (hunk 1 above)
Comment (aquamatthias) [reply fixbackend/organizations/router.py:171]: why would we ever provide the external id? Isn't this considered a secret?

File: fixbackend/organizations/router.py (hunk 4 above)
Comment (meln1k) [reply fixbackend/organizations/router.py:191]: This one will point to the cloud formation template, which is not there yet. You probably talking about the callback, let me add it here.

File: migrations/versions/2023-09-20T09:32:21Z_add_externalid_to_org.py (hunk 3 above)
Comment (meln1k) [reply migrations/versions/2023-09-20T09:32:21Z_add_externalid_to_org.py:30]: why won't we? if we want to revert the db to the previous state, we don't want to keep the additional data.

**Pull request merged**